chaos_wrapper.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+function rollback() {
+    git checkout master
+    git checkout HEAD^
+}
+
+/root/.virtualenvs/chaos/bin/python chaos.py || rollback
```

Comment:
That would be an amazing attack vector! I didn't think about that. Maybe revert up to and including the last chaosbot commit?